modules/nf-scil/betcrop/synthbet/meta.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: "betcrop_synthbet"
+description: Perform brain extraction using mri_synthstrip on t1 image
+keywords:
+  - anatomical image
+  - BET
+  - freesurfer
+tools:
+  - "Freesurfer":
+      description: "Software package for the analysis and visualization of structural and functional neuroimaging data."
+      homepage: "https://surfer.nmr.mgh.harvard.edu/fswiki"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+
+  - t1:
+      type: file
+      description: Nifti image volume to perform BET.
+      pattern: "*.{nii,nii.gz}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+
+  - bet_t1:
```

Comment:
Same as above, `bet_t1` -> `bet_image`. Same thing in the description below (also in the description of the `- mask:` output).